src/ts/services/TyperighterTelemetryAdapter.ts

```diff
@@ -11,17 +11,23 @@ import {
   ITyperighterTelemetryEvent,
   TYPERIGHTER_TELEMETRY_TYPE
 } from "../interfaces/ITelemetryData";
-import { UserTelemetryEventSender } from "@guardian/user-telemetry-client";
+import { IUserTelemetryEvent, UserTelemetryEventSender } from "@guardian/user-telemetry-client";
 import { IMatch } from "..";
 import { MatchType } from "../utils/decoration";
 
 class TyperighterTelemetryAdapter {
   constructor(
     private telemetryService: UserTelemetryEventSender,
     private app: string,
-    private stage: string
+    private stage: string,
+    private tags?: IUserTelemetryEvent["tags"],
   ) {}
 
+  // used by flexible-content to update tags that are only known after the telemetry adaptor has been initialised
```

Comment:
Minor – we probably shouldn't assume the consumer here (we dream of Typerighter in Fronts, maybe marketing tools), but fair to say a comment of some sort still welcome!